tsconfig.json

```diff
@@ -10,7 +10,9 @@
     "alwaysStrict": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
-    "strictNullChecks": true
+    "strictNullChecks": true,
+    "baseUrl": "./",
+    "paths": { "plotly.js-basic-dist-min": ["node_modules/@types/plotly.js"] },
```

Comment:
https://stackoverflow.com/questions/39084438/how-to-import-plotly-js-into-typescript